modApiServer/src/org/aion/api/server/Api.java

```diff
@@ -50,7 +50,20 @@
 
     private final AccountManager ACCOUNT_MANAGER = AccountManager.inst();
     private final Compiler solc = Compiler.getInstance();
-    protected final AionPendingStateImpl pendingState = AionPendingStateImpl.inst();
+    protected final AionPendingStateImpl pendingState;
+
+    // This is the constructor that should always be used, unless testing
+    Api() {
+        pendingState = AionPendingStateImpl.inst();
+    }
+
+    // Only for testing purposes
+    Api(boolean test) {
```

Comment:
Also, my p2p PR update the mockito version, you can just mock the final class without changing the code. please considering to remove the pendingState.

Reply:
i wasn't able to mock the AionPendingStateImpl.inst() because it's a static method, and mockito doesn't seem to support mocking statics.